SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ViewDeltaTest.cpp

```diff
@@ -1,7 +1,134 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
+#include "OpsUtils.h"
+#include "SpatialViewUtils.h"
 #include "Tests/TestDefinitions.h"
 
 #include "SpatialView/ViewDelta.h"
 
 #define VIEWDELTA_TEST(TestName) GDK_TEST(Core, ViewDelta, TestName)
+
+using namespace SpatialGDK;
+
+VIEWDELTA_TEST(GIVEN_empty_view_WHEN_add_remove_THEN_get_empty_view_and_delta)
```

Comment:
Could you include the base case (just a single add) and check as well as the add-remove and add-remove-add cases?